common/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java

```diff
@@ -524,6 +524,7 @@ public enum ErrorMsg {
   CTLF_MISSING_STORAGE_FORMAT_DESCRIPTOR(20021, "Failed to find StorageFormatDescriptor for file format ''{0}''", true),
   PARQUET_FOOTER_ERROR(20022, "Failed to read parquet footer:"),
   PARQUET_UNHANDLED_TYPE(20023, "Unhandled type {0}", true),
+  ORC_FOOTER_ERROR(20024, "Failed to read orc footer:"),
```

Comment:
Well the colon is there in ``PARQUET_FOOTER_ERROR`` but I am not sure why it is so, when this exception is thrown, post the colon it is the trace, should have been a period according to me, but let it be to be in sync with PARQUET_FOOTER_ERROR. But just out of curiosity if you know the reason behind it do let me know

Reply:
I just follow the PARQUET_FOOTER_ERROR, and we can let @jfsii give a explanation. :)

Reply:
The source for the change was a comment on a review I had on the patch:
"nit: add a ":" such that the exception is printed after the delimiter"

I could have pushed back on the comment.